src/views/UpdateLocusGeneDiseaseView.vue

```diff
@@ -646,10 +669,9 @@ export default {
       :stableId="stableID"
     />
     <PublishModal @publish="saveAndPublishEntry" />
-    <AlertModal
-      modalId="publications-input-alert-modal"
-      alertText="The data you have input under Publications, Phenotypic Features, Variant Types, Variant Description, and Mechanism Evidence will be lost. Are you sure you want to proceed?"
-      @confirm-click-handler="fetchPublications"
+    <RemovePublicationModal
+      :pmidList="Object.keys(publications || {})"
```

Comment:
```suggestion
      :pmidList="Object.keys(previousInput.publications)"
```

Reply:
If i do it without the initializiation of an empty array, there is an error in the console. Thats why i did it this way

Reply:
```suggestion
      :pmidList="Object.keys(previousInput?.publications || {})"
```

Reply:
understood the problem, ok then, change it like above suggestion